src/apps/distributed_efforts/services/rating_network_pairer.py

```diff
@@ -101,7 +102,7 @@ def _choose_opponent(self, reference_network):
 
         # Vary the reference net's log gamma so that networks that are uncertain will play a greater variety of opponents
         # based on that uncertainty
-        ref_net_log_gamma = reference_network.log_gamma + np.random.normal() * reference_network.log_gamma_uncertainty
+        ref_net_log_gamma = reference_network.log_gamma + np.random.normal() * reference_network.log_gamma_uncertainty * self.current_run.rating_game_variability_scale
```

Comment:
This scale is supposed to increase the "deviation" where network can be chosen ?

Reply:
Answered in migration